web/concrete/attributes/select/form.php

```diff
@@ -83,8 +83,8 @@
 					}
 				},
 				<? if ($akSelectAllowMultipleValues) { ?>
-					tokenSeparators: ['|'],
-					separator: "|",
+					tokenSeparators: ['<?php echo \Config::get('app.attributes.select.multiple.separator'); ?>'],
```

Comment:
It's always safer to use
```
tokenSeparators: [<?php echo json_encode($x); ?>]
```

instead of

```
tokenSeparators: ['<?php echo $x; ?>']
```